#GPS_Data
float32 curr_lat
float32 curr_long
int64 elapsed_time
int16 sats
float32 speed_val
float32 heading
#Diagnostics data
int16 bin_fullness
int16 battery
bool pickup_state
float32 l_motor
float32 r_motor
